scenarios/cut_in/agent_prefabs.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from smarts.core.agent import Agent, AgentSpec
+from smarts.core.agent_interface import AgentInterface
+from smarts.core.controllers import ActionSpaceType
+from smarts.zoo.registry import register
+
+
+class PoseBoidAgent(Agent):
+    def act(self, obs):
+        return {vehicle_id: self._single_act(obs_) for vehicle_id, obs_ in obs.items()}
+
+    def _single_act(self, obs):
+        wp = obs.waypoint_paths[0][:5][-1]
+        dist_to_wp = np.linalg.norm(wp.pos - obs.ego_vehicle_state.position[:2])
+        target_speed = 5  # m/s
+        return np.array([*wp.pos, wp.heading, dist_to_wp / target_speed])
+
+
+class TrajectoryBoidAgent(Agent):
```

Comment:
What do you think about making these two agents part of the official zoo? We tend to use them frequently.

Reply:
Oh, I should have mentioned this cut_in scenario will be removed before I merge this PR. This was copied from the straight scenario, and actually I didn't use these two agents. :smile:

Reply:
oh hehe